tests/main/remote-home/task.yaml

```diff
@@ -117,38 +122,44 @@ prepare: |
     tests.cleanup defer systemctl restart snapd.service
     tests.cleanup defer systemctl reset-failed snapd.service snapd.socket
 
+    # leave a canary file
+    touch /var/home/test-remote/hello
     # Mount the remote filesystem at /home/test-remote.
     case "$FS" in
       nfs)
-        mount -t nfs localhost:/var/home/test-remote /home/test-remote -o nfsvers=3,proto=tcp
+        mount -t nfs localhost:/var/home/test-remote /home/test-remote #-o nfsvers=3,proto=tcp
```

Comment:
Is this deliberate? Either get rid of it or don't comment it out like that.

Reply:
deliberate as the options were unnecessary, but forgot to drop it.